src/screens/CreatePost/elements/SheetCloseBtn.js

```diff
@@ -23,14 +21,14 @@ const SheetCloseBtn = ({backRef, goBack, continueToEdit, isAnonimity = false}) =
       <View style={styles.container}>
         <Text style={styles.header}>Are you sure?</Text>
         <Gap style={styles.gap(30)} />
-        <Button onPress={continueToEdit} style={{backgroundColor: dynamicColors.primary}}>
+        <Button onPress={continueToEdit}>
           <Text>Continue editing</Text>
         </Button>
         <Gap style={styles.gap(10)} />
         <Button
           onPress={goBack}
-          style={{backgroundColor: COLORS.porcelain}}
-          textStyling={{color: COLORS.black}}>
+          styles={{backgroundColor: colors.porcelain}}
+          textStyling={{color: colors.black}}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color scheme has been changed from dynamic colors to static ones. This might affect the user experience if the application supports themes or dark mode. Please ensure that this change aligns with your design guidelines and doesn't degrade the user experience.
```diff
-          style={{backgroundColor: COLORS.porcelain}}
-          textStyling={{color: COLORS.black}}>
+          styles={{backgroundColor: colors.porcelain}}
+          textStyling={{color: colors.black}}>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->